graph/load.go

```diff
@@ -68,7 +68,7 @@ func (s *TagStore) CmdLoad(job *engine.Job) engine.Status {
 
 		for imageName, tagMap := range repositories {
 			for tag, address := range tagMap {
-				if err := s.Set(imageName, tag, address, true); err != nil {
+				if err := s.SetLoad(imageName, tag, address, true, job.Stdout); err != nil {
```

Comment:
@icecrime it's a warning message not an error. We still want the `docker load` to go through.